thirdeye/thirdeye-frontend/tests/acceptance/self-serve-import-metric-test.js

```diff
@@ -0,0 +1,168 @@
+import $ from 'jquery';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { selfServeConst } from 'thirdeye-frontend/tests/utils/constants';
+import { visit, fillIn, click, currentURL, triggerKeyEvent } from '@ember/test-helpers';
+
+module('Acceptance | import metric', function(hooks) {
+  setupApplicationTest(hooks);
+
+  const importLinkText = 'Import a Metric From InGraphs';
```

Comment:
I recall you created a constants file for test. Perhaps some of those should go there?

Reply:
Yes. For now I'm keeping the constants file exclusively for element selectors. If the team wants to add all vars, we can consider, but this seems cleaner.